### PR TITLE
cleanup acme -> e3sm references in rst documentation

### LINKED ISSUE #2379: acme still referenced in some of the scripts/Tools files
I found these references to ACME in cime/scripts/Tools:

'grep -r acme *
bless_test_results:    > {0} -r /home/jenkins/acme/scratch/jenkins -b next --hist-only
compare_test_results:    > {0} -r /home/jenkins/acme/scratch/jenkins -b next
pointclm/pointCLM.py:#PTCLMfiledir = csmdir+'/scripts/acme/pointclm/PTCLM_files/'
pointclm/pointCLM.py:os.system('mkdir -p '+csmdir+'/cime/scripts-acme/pointclm/temp')
pointclm/pointCLM.py:              +csmdir+'/cime/scripts-acme/pointclm/temp/clm_params.'+pftphys_stamp+'.'+ \
pointclm/pointCLM.py:os.system('chmod u+w ' +csmdir+'/cime/scripts-acme/pointclm/temp/clm_params.'+pftphys_stamp+'.'+ \
pointclm/pointCLM.py:    pftfile = csmdir+'/cime/scripts-acme/pointclm/temp/clm_params.'+pftphys_stamp+'.'+casename+'.nc'
pointclm/pointCLM.py:              +csmdir+'/cime/scripts-acme/pointclm/temp/CNP_parameters_'+CNPstamp+ \
pointclm/pointCLM.py:os.system('chmod u+w ' +csmdir+'/cime/scripts-acme/pointclm/temp/CNP_parameters_'+CNPstamp+ \
pointclm/pointCLM.py:os.system('mv '+csmdir+'/cime/scripts-acme/pointclm/temp/*'+casename+'* ' \
pointclm/pointCLM.py:   os.system('mv '+csmdir+'/cime/scripts-acme/pointclm/temp/domain*'+options.site+'* ' \
pointclm/pointCLM.py:        output = open(csmdir+'/cime/scripts-acme/pointclm/temp/ensemble_run'+numst[1:]+'.pbs','w')
pointclm/pointCLM.py:                              '/cime/scripts-acme/pointclm/temp/mynodefile'+ngst[1:]+'\n')
pointclm/pointCLM.py:                                 csmdir+'/cime/scripts-acme/pointclm/temp/mynodefile'+ngst[1:]+ \
pointclm/pointCLM.py:                                 ' '+orig_dir+'/acme.exe > ccsm_log.txt &\n')
pointclm/pointCLM.py:                os.system('qsub '+csmdir+'/cime/scripts-acme/pointclm/temp/ensemble_run'+ \
pointclm/pointCLM.py:                              numst[1:]+'.pbs > '+csmdir+'/cime/scripts-acme/pointclm/temp/jobinfo')
pointclm/pointCLM.py:                myinput = open(csmdir+'/cime/scripts-acme/pointclm/temp/jobinfo')
pointclm/pointCLM.py:                              '/cime/scripts-acme/pointclm/temp/ensemble_run'+ \
pointclm/pointCLM.py:                              numst[1:]+'.pbs > '+csmdir+'/cime/scripts-acme/pointclm/temp/jobinfo')
pointclm/makepointdata.py:            metcmd = 'python '+csmdir+'/cime/scripts-acme/makemetdata.py' \
pointclm/makepointdata.py:os.system('mkdir -p '+options.csmdir+'/cime/scripts-acme/pointclm/temp')
pointclm/makepointdata.py:domainfile_new = options.csmdir+'/cime/scripts-acme/pointclm/temp/' \
pointclm/makepointdata.py:    surffile_new  = options.csmdir+'/cime/scripts-acme/pointclm/temp/surfdata_'+str(numxpts)+'x' \
pointclm/makepointdata.py:    surffile_new =  options.csmdir+'/cime/scripts-acme/pointclm/temp/surfdata_'+str(numxpts)+'x'+str(numypts)+'pt_'+ \
pointclm/makepointdata.py:        pftdyn_new = options.csmdir+'/cime/scripts-acme/pointclm/temp/' \
pointclm/makepointdata.py:        pftdyn_new = options.csmdir+'/cime/scripts-acme/pointclm/temp/' \
xmlconvertors/config_pes_converter.py:    CIME2: cime/machines-acme/config_pes.xml
xmlconvertors/config_pes_converter.py:    CIME5: config/acme/allactive/config_pesall.xml
xmlconvertors/grid_xml_converter.py:    CIME5: config/acme/config_grids.xml
xmlconvertors/grid_xml_converter.py:#    CIME5: config/acme/config_grids.xml
xmlconvertors/grid_xml_converter.py:                logger.info("config/acme/config_grids.xml. These nodes")'

### PR DESCRIPTION
**Pull request by bertinia**
cleaned up old acme references in the doc/source rst files and scripts/Tools and replacing it with e3sm.

Test suite: N/A
Test baseline: N/A
Test namelist changes: N/A
Test status: [bit for bit, roundoff, climate changing] bit for bit

Fixes  #2379

User interface changes?: None

Update gh-pages html (Y/N)?: Y

### CODE REVIEW

**Comment by rljacob** [comment]
** PLEASE NOTE ** When editing the cime/doc/source/*.rst files, please check that the html generates
correctly before issuing a PR by running the following:

>cd cime/doc
>make clean
>make api
>make html

There are still some sphinx warnings and errors associated with the docstrings in some 
of the python scripts. These will be addressed in another PR once we have a release branch.

File: doc/source/users_guide/testing.rst [hunk 1]
```diff
@@ -220,19 +220,19 @@ To run two tests::
...
+  ./create_test e3sm_developer ^SMS.f19_f19.A
 
 See create_test -h for the full list of options
 
 Interpreting test output is pretty easy, looking at an example::
 
   % ./create_test SMS.f19_f19.A
 
-  Creating test directory /home/jgfouca/acme/scratch/SMS.f19_f19.A.melvin_gnu.20170504_163152_31aahy
+  Creating test directory /home/jgfouca/e3sm/scratch/SMS.f19_f19.A.melvin_gnu.20170504_163152_31aahy
```
Comment (rljacob) [inline doc/source/users_guide/testing.rst]: This was one of the instances that was ok to leave as acme.  But its just an example and it doesn't matter if the dir path is real or not so don't redo it.

**Comment by bertinia** [comment]
Thanks @rljacob - can you merge these changes and I'll update the gh-pages with the html?

**Comment by rljacob** [comment]
Merged

**Pull request merged**